default/python/AI/freeorion_tools/timers.py

```diff
@@ -123,4 +100,4 @@ def end(self):
             self._write(''.join(row))
         self.timers = []  # clear times
 
-Timer = LogTimer if USE_TIMERS else DummyTimer
+FOTimer = FOLogTimer if USE_TIMERS else DummyTimer
```

Comment:
Maybe leave `Timer` name as it was? And rename `commons.Timer`  to `BaseTimer`

I am against `FO` prefix. It does not explain anything every part of code is `FO`. Use `AI`